system-tests/test-binary/yarn_pnp_spec.ts

```diff
@@ -0,0 +1,12 @@
+import systemTests from '../lib/system-tests'
+
+describe('yarn PnP 4.1.1', () => {
+  // @see https://github.com/cypress-io/cypress/issues/27947
+  systemTests.it('can run the test via resolving all dependencies in the support file', {
```

Comment:
since we need to intall the dependencies with yarn 4, this needs to be a binary system test